jdaviz/configs/cubeviz/plugins/unified_slider/unified_slider.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+from traitlets import Bool, Float, Unicode, observe
+
+from jdaviz.core.events import AddDataMessage, AddViewerMessage
+from jdaviz.core.registries import tool_registry, viewer_registry
+from jdaviz.core.template_mixin import TemplateMixin
+from jdaviz.utils import load_template
+
+from glue.external.echo import add_callback, remove_callback, delay_callback
+from glue_jupyter.bqplot.image import BqplotImageView
+
+__all__ = ['UnifiedSlider']
+
+
+@tool_registry('g-unified-slider')
+class UnifiedSlider(TemplateMixin):
+    template = load_template("unified_slider.vue", __file__).tag(sync=True)
+    slider = Float().tag(sync=True)
+    min_value = Float(0).tag(sync=True)
+    max_value = Float(100).tag(sync=True)
+    linked = Bool(True).tag(sync=True)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self._watched_viewers = []
+
+        # Listen for add data events. **Note** this should only be used in
+        #  cases where there is a specific type of data expected and arbitrary
+        #  viewers are not expected to be created. That is, the expected data
+        #  in _all_ viewers should be uniform.
+        self.session.hub.subscribe(self, AddDataMessage,
+                                   handler=self._on_data_added)
+
+    @observe("linked")
+    def _on_linked_changed(self, event):
+        for viewer in self._watched_viewers:
+
+            if not event['new']:
+                remove_callback(viewer.state, 'slices',
```

Comment:
In general I find it easier to do ``viewer.state.remove_callback`` and same with ``add_callback``